pom.xml

```diff
@@ -42,6 +42,9 @@
     <jenkins-test-harness.version>2.38</jenkins-test-harness.version>
     <scm-api.version>2.2.6</scm-api.version>
     <git.version>3.8.0</git.version>
+    <!-- upgrading to parent 3.15 introduce a new version which is VERY MORE MORE restrictive so skip it
+    as this need a lot of changes.... -->
+    <access-modifier-checker.failOnError>false</access-modifier-checker.failOnError>
```

Comment:
@olamy This should always be true, no? We do not want to disable this check.

Reply:
ohh gloupss I changed the code but not this one.
Fixed!

Reply:
Well we have more problems now....
jenkins.branch.OrganizationFolder from branch-api is marked as @Restricted({NoExternalUse.class}) and we use it.
As it's external, I cannot move it.